invenio_drafts_resources/resources/drafts/__init__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```

Comment:
Minor: This should be consistent with invenio-records-resources, so either 1) not move the files or 2) also perform the move in invenio-records-resources.

Reply:
Move also done in records-resources :heavy_check_mark: